kuberun/eventing/audit-pubsub/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
```

Comment:
If these samples are related to Cloud Run, would you mind moving them to the `run/` directory in this repo?